### DISCUSSION TITLE
How to change sidebar color

### DISCUSSION THREAD

**Discussion by wisliy** [orchidsoftware/platform#1758] [Q&A] [answered]
The documentation shows how to change the logo and name on the platform, to match the visual style of my brand. Is there a way to safely customize the sidebar color too?

I've created a custom dashboard template by copying the `dashboard.blade.php` from `vendor/orchid/platform/resouces/views` to /resources/views/brand, in the same way of header and footer, but it doesn't work.

`
  'template' => [
    'header' => 'brand.header',
    'footer' => 'brand.footer',
    'dashboard' => 'brand.dashboard',
  ],
`

I also directly edited the original dashboard template in `vendor/orchid/platform/resouces/views`, including a custom CSS selector called `bg-custom` at `<div class="aside col-xs-12 col-md-2 bg-dark bg-custom">` and it worked, but as I'm using github actions to deploy the application and installing the composer dependencies inside the actions, it overriding my customization.

Thanks in advance.

**Comment by EeKay** [comment, marked as answer]
I've been using Orchid for a while now (for a customer project; a local school that needs a nice web app) and I really love it (thanks @tabuna  for creating this 🙌🏻!)

As far as I've seen, the "proper" way to change CSS is to:

1. make changes in the .sass files located under the subdirectory vendor\orchid\platform\resources\sass
2. compile the sass files into css
3. add the resulting orchid.css to subdir \public\css\orchid

If I've interpreted it correctly, the app.blade.php file (under vendor\orchid\platform\resources\views\app.blade.php states that it will run mix on the file "public\css\orchid\orchid.css" if it exists, else it uses orchid_mix to run its original orchid.css file..

For completeness sake, here's the code in app.blade.php file:

`@if(file_exists(public_path('/css/orchid/orchid.css')))
    <link rel="stylesheet" type="text/css" href="{{  mix('/css/orchid/orchid.css') }}">
@else
    <link rel="stylesheet" type="text/css" href="{{  orchid_mix('/css/orchid.css','orchid') }}">
@endif
...`

@tabuna could you verify that this is the proper (or an intended) way to customize styling for orchid based projects?

**Reply by tabuna** [reply]
Hi @EeKay, no source build is suggested (but possible), because updates will not come. The safest way is to submit a PR for change or include your own CSS to override styles.

**Comment by alex-griffiths** [comment]
Is there any other information about how to correctly update CSS at this stage? Currently I've hade to link the CSS file and now I'm having to find and replace specific colours. I'm really hoping there's a better way to do this.

**Reply by wisliy** [reply]
Hi, @alex-griffiths!

I did this by copying the `dashboard.blade.php` file from `vendor/orchid/platform/resources/views` to `resouces/views/vendor/platform` and added the custom css class to `<div class="aside col-xs-12 col-md-2 bg-dark **bg-custom**">`.
